addons/stock_account/models/stock_move.py

```diff
@@ -473,38 +467,37 @@ def _account_entry_move(self, qty, description, svl_id, cost):
         journal_id, acc_src, acc_dest, acc_valuation = self._get_accounting_data_for_valuation()
         # Create Journal Entry for products arriving in the company; in case of routes making the link between several
         # warehouse of the same company, the transit location belongs to this company, so we don't need to create accounting entries
+        am_vals = []
         if self._is_in():
             if self._is_returned(valued_type='in'):
-                self.with_company(company_to)._create_account_move_line(acc_dest, acc_valuation, journal_id, qty, description, svl_id, cost)
+                am_vals.append(self.with_company(company_to)._prepare_account_move_vals(acc_dest, acc_valuation, journal_id, qty, description, svl_id, cost))
```

Comment:
if `_prepare_account_move_vals` can return None (in case there is not `move_lines`), the append will add None. And I think It will be problematic in the create after, No ?